<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWindow</class>
 <widget class="QMainWindow" name="ProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableDataSources"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>1. Data Sources</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>2. Track Definition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
             <item>
              <widget class="QComboBox" name="newTrackSourceCombo"/>
             </item>
             <item>
              <widget class="QPushButton" name="newTrackButton">
               <property name="text">
                <string>Create Track</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="editTrackButton">
             <property name="text">
              <string>Edit Track</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="topBottomWizard">
          <property name="text">
           <string>Top/Bottom Wizard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>3. Objects</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="renderEntitiesTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="alignmentGroupBox">
             <property name="title">
              <string>4. Data Alignment</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Lap: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="lapSpinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QRadioButton" name="entryRadio">
                  <property name="text">
                   <string>Entry</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="exitRadio">
                  <property name="text">
                   <string>Exit</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="customAlignmentCheckBox">
                <property name="text">
                 <string>Custom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="customAlignmentTableWidget"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="exportGroupBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>5. Export</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
                <item>
                 <widget class="QLineEdit" name="exportFileLineEdit"/>
                </item>
                <item>
                 <widget class="QPushButton" name="browseExportButton">
                  <property name="text">
                   <string>Browse</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="exportButton">
                <property name="text">
                 <string>Export</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopButton">
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_Recent_Project">
     <property name="title">
      <string>Load Recent Project</string>
     </property>
     <addaction name="actionProject1"/>
     <addaction name="actionProject2"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_as"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="menuLoad_Recent_Project"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Sources"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuPreview_Resolution">
     <property name="title">
      <string>Preview Resolution</string>
     </property>
     <addaction name="action960_x_540"/>
     <addaction name="action1280_x_720"/>
     <addaction name="action1920_x_1080"/>
    </widget>
    <addaction name="actionShow_Render_Preview"/>
    <addaction name="menuPreview_Resolution"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_as">
   <property name="text">
    <string>Save Project as...</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project...</string>
   </property>
  </action>
  <action name="actionShow_Track_Editor">
   <property name="text">
    <string>Show Track Editor</string>
   </property>
  </action>
  <action name="actionShow_Render_Preview">
   <property name="text">
    <string>Show Render Preview</string>
   </property>
  </action>
  <action name="actionImport_Sources">
   <property name="text">
    <string>Import Sources...</string>
   </property>
  </action>
  <action name="actionProject1">
   <property name="text">
    <string>Project1</string>
   </property>
  </action>
  <action name="actionProject2">
   <property name="text">
    <string>Project2</string>
   </property>
  </action>
  <action name="action1280_x_720">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1280 x 720</string>
   </property>
  </action>
  <action name="action1920_x_1080">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1920 x 1080</string>
   </property>
  </action>
  <action name="action960_x_540">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>960 x 540</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
